prow/jobs/jobs_config.yaml

```diff
@@ -26,4 +26,12 @@ aws_services:
 # List of AWS services onboarded for soak testing, executing in their Prow
 # cluster on every release
 soak_test_on_release_service_names:
-- ecr
\ No newline at end of file
+- ecr
+
+# list of AWS service controllers to be auto-generated on every release of ACK
+# code-generator
+auto_generate_on_codegen_release_service_names:
```

Comment:
Why not use the list from e2e tests? We should assume every team wants this functionality tbh

Reply:
Adding a todo to eventually move to e2e tests.

For the very first release, I just wanted to have less services & see how it works.
Fix any issues that arise with the prowjob and then reuse the list for e2e tests.

Do not want to start off by adding all the services.